public/app/features/commandPalette/values.ts

```diff
@@ -1,3 +1,4 @@
+export const EXTENSIONS_PRIORITY = 6;
```

Comment:
It's worth considering what this priority should be; this puts extensions right at the top which is probably a bit extreme, maybe somewhere in the middle makes sense?